docs/source/API/core/execution_spaces.rst

```diff
@@ -308,6 +308,8 @@ Functions
 
 * ``const char* name() const;``: *Returns* the label of the execution space instance.
 
+* ``bool is_running() const;``: *Returns* whether the execution space has completed all kernels submitted to it or if any of the kernels submitted are still running.
```

Comment:
So, if this variable evaluates to True, then the kernels associated with this execution space are still running?  If False, then they have (successfully) completed?  Maybe just refine the wording a bit so it will be clear what True and False connote.

Reply:
Reworded.